src/ci/docker/x86_64-gnu-tools/checkregression.py

```diff
@@ -5,7 +5,7 @@
 import json
 
 # Regressions for these tools does not cause failure
-REGRESSION_OK = ["rustc-guide"]
+REGRESSION_OK = ["rustc-guide", "miri"]
```

Comment:
To sync the list with `checktools.sh`, this should also include `"embedded-book"`.

Reply:
@mark-i-m I think this one is pretty clear: the list here should be the same as the `nightly` tools in `checktools.sh`.

Reply:
But that list does _not_ include `rustc-guide`?

Reply:
Oh, because you removed it. It does in master...

Reply:
Well I don't know how much sense it makes to exclude tools there nor what consequences it even has, but @kennym agreed that `embedded-book` should get the same treatment as Miri.

Reply:
Ok, added it.